py3-graph-tool.yaml

```diff
@@ -78,15 +82,17 @@ subpackages:
       - uses: autoconf/configure
         with:
           opts: |
-            --enable-openmp \
             --includedir=/usr/lib/cairomm-1.16 \
-            --disable-debug \
-            --disable-dependency-tracking \
```

Comment:
Found that behavior here (sans dependency tracking which should be enabled) was default (and that some of these flags aren't valid)